build.gradle

```diff
@@ -55,18 +89,18 @@ task release {
 			"the fact that subprojects will appropriately define a release task " +
 			"themselves if they have any release-related activities to perform"
 
-	// Force to release with JDK 8. Releasing with JDK 11 is not supported yet:
-	// - the hibernate-orm-modules tests do not run due to an issue with the ASM version currently used by Gradle
+	// Force to release with JDK 8. It used to not work on JDK11 because of the hibernate-orm-modules module,
+	// but this limitation might be resolved now that this module has been deleted?
```

Comment:
I have no idea if this matters, but Hibernate ORM 5.3 still has the hibernate-orm-modules tests.

In any case, JDK 1.8 will be used for releasing.

Reply:
I'll update the comment in a new commit.